frontend/src/features/documents/documentUpload/DocumentUploadView.tsx

```diff
@@ -22,6 +24,7 @@ interface IDocumentUploadViewProps {
   documentTypes: Api_DocumentType[];
   isLoading: boolean;
   mayanMetadata: Api_Storage_DocumentTypeMetadataType[];
+  documentMetadata: any;
```

Comment:
what is this variable used for? If we should avoid having 'any' as much as posible

Reply:
At this point the document would have no metadata attached

Reply:
removed